Content.Server/Disposal/Unit/Components/DisposalUnitComponent.cs

```diff
@@ -240,6 +239,37 @@ protected override void Activate(IEntity user, DisposalUnitComponent component)
             }
         }
 
+        [Verb]
+        private sealed class EjectVerb : Verb<DisposalUnitComponent>
+        {
+            public override bool AlternativeInteraction => true;
+
+            protected override void GetData(IEntity user, DisposalUnitComponent component, VerbData data)
+            {
+                data.Visibility = VerbVisibility.Invisible;
+
+                if (!EntitySystem.Get<ActionBlockerSystem>().CanInteract(user) ||
+                    component.ContainedEntities.Contains(user))
+                {
+                    return;
+                }
+
+                // Only show verb if actually containing any entities.
```

Comment:
can potentially be used to metagame information about disposal contents but probably not an actual issue

Reply:
I'll test that it's properly blocked, but except for some global verbs like examine, most verbs should only show up if you're in range & unobstruced. Also, I think the disposal unit blinks if it has non-zero conents anyways, so I don't think this reveals any new information.

Reply:
It is blocked when out of range, but there was a seperate issue with duplicate localization strings, so I just updated those